security/pkg/stsservice/tokenmanager/google/tokenexchangeplugin.go

```diff
@@ -382,8 +379,6 @@ func (p *Plugin) constructGenerateAccessTokenRequest(fResp *federatedTokenRespon
 	if pluginLog.DebugEnabled() {
 		reqDump, _ := httputil.DumpRequest(req, true)
 		pluginLog.Debugf("Prepared access token request: \n%s", string(reqDump))
-	} else {
```

Comment:
reduce spammy logs that do not have any value; we don't care that we created a http.Request object we care that we sent the request which we already log appropriately.